### PR TITLE
Introduced SkippedIterable and SkippedIterator

### PR DESCRIPTION
**Pull request by smallcreep**
As per #113

### CODE REVIEW

**Comment by 0crat** [comment]
@smallcreep I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

File: src/main/java/org/cactoos/list/SkippedIterator.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+     * @param skip Count skip elements
+     */
+    public SkippedIterator(final Iterator<T> iterator, final int skip) {
+        this.iterator = iterator;
+        this.skip = skip;
+    }
+
+    @Override
+    public boolean hasNext() {
+        if (this.skip > 0) {
```
Comment (alex-semenyuk) [inline src/main/java/org/cactoos/list/SkippedIterator.java]: This check is already existed below.

Comment (smallcreep) [inline src/main/java/org/cactoos/list/SkippedIterator.java]: Yea I know. I dleted it.

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**